### DISCUSSION TITLE
Basic Getting Started

### DISCUSSION THREAD

**Discussion by 0xlee** [HomeSmartMesh/sdk-hsm-sensortag#13] [Q&A] [answered]
Hi,

I received your product, thanks!

But I am some missing getting started documentation. The blog didn't help me either, so I have to ask here some basic questions.

- Does the device come with preflashed firmware? If yes, which one?
- If I want to flash with one of your samples, I assume that I have to connect some pins to the board. But the question is: what kind of connector do I need? A segger j-link? Or some other stuffs having RX/TX?

I am quite new to hardware and zephyr. I have experience in frontend/backend software engineering and I only played with some hardware devices having USBs.

Thanks!

Best Regards
Dongha

**Comment by wassfila** [comment, marked as answer]
Hi @0xlee ,
it is not clear to me what did you receive and from whom ? If you could post links so that I understand.

In this website I created, I provided guides to understand the project : https://www.homesmartmesh.com/docs/microcontrollers/nrf52/thread_sensortag/

I have to admit this project should not be the kind of projects you start with, always start with an existing dev Kit such as https://www.nordicsemi.com/Products/Development-hardware/nrf52840-dk

There you have everything with right connectors and guides from manufacturer so that nothing can go wrong.

Once you are familiar with nRF52, then for Zephyr you should also start with hello world simple examples 
this is a good starting point : https://docs.zephyrproject.org/2.3.0/getting_started/index.html

You can even then try examples for one of the devkit.

This project is to explore OpenThread Stack which can also run on existing ready HW devices which I recommend.

Once familiar with all those steps, you can initiate a custom made hw, this one or even design your own.

Hope this helps
